Library/Homebrew/extend/os/linux/requirements/java_requirement.rb

```diff
@@ -0,0 +1,31 @@
+require "language/java"
```

Comment:
Note to self: remember to fix that commit message, because it's rather incorrect at this point.